plugins/dependencies/src/main/kotlin/dependencies/ReactiveX.kt

```diff
@@ -0,0 +1,34 @@
+@file:Suppress("PackageDirectoryMismatch", "SpellCheckingInspection", "unused", "MemberVisibilityCanBePrivate")
+
+import de.fayard.refreshVersions.core.DependencyNotation
+import de.fayard.refreshVersions.core.DependencyNotationAndGroup
+import org.gradle.api.Incubating
+import org.gradle.kotlin.dsl.IsNotADependency
+
+@Incubating
+object ReactiveX : IsNotADependency {
+
+    val rxJava = DependencyNotation(group = "io.reactivex", name = "rxjava")
+
+    val rxAndroid = DependencyNotation(group = "io.reactivex", name = "rxandroid")
+
+    val rxKotlin = DependencyNotation(group = "io.reactivex", name = "rxkotlin")
```

Comment:
That's RxJava1, right? I'm pretty sure it's no longer updated, so I don't see much value in adding it into refreshVersions.

Reply:
I initially didn't have it in there, but then I saw the RxJava1 adapter in Square.Retrofit2, so I figured I would add it. I agree though with you though so I'll remove it